src/gui/src/scriptWidget.cpp

```diff
@@ -139,11 +139,11 @@ ScriptWidget::ScriptWidget(QWidget* parent)
 // the logging, so, for reports, we use a buffer.
 void ScriptWidget::flushReportBufferToOutput()
 {
+  std::lock_guard guard(reporting_);
```

Comment:
I'm assuming you meant to include this

Reply:
Yeah, it was in the list and small enough to be quick